docs/general/builtins/scalar-functions.rst

```diff
@@ -2785,6 +2785,46 @@ function supports arrays of any of the :ref:`primitive types
     SELECT 1 row in set (... sec)
 
 
+.. _scalar-array_position:
+
+``array_position(anycompatiblearray, anycompatible [, integer ] ) → integer``
+-----------------------------------------------------------------------------
+
+The ``array_position`` function returns the position of the first
+occurrence of the second argument in the ``array``, or ``NULL`` if it's not
+present. If the third argument is given, the search begins at that position.
+The third argument is ignored if it's null or not in within the ``array``
+range. It is also possible to search for ``NULL`` values.
```

Comment:
```suggestion
present. If the third argument is given, the search begins at that position.
The third argument is ignored if it's null. If not within the ``array`` range,
``NULL`` is returned. It is also possible to search for ``NULL`` values.
```

Reply:
should be something like this, according to https://github.com/crate/crate/pull/11989#discussion_r782148204

Reply:
👍